GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/GameData.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+
+[Serializable]
+public class GameData
+{
+    public string carInUseName;
+    public bool[] carIndexToOwnership;
+    public int coinsOwned;
```

Comment:
Don't let other class to access this object, make a getter method: https://codeasy.net/lesson/properties